packages/shared/src/relay/useGetPreviewImages.ts

```diff
@@ -6,25 +6,37 @@ import { useGetPreviewImagesSingleFragment$key } from '~/generated/useGetPreview
 import { useGetPreviewImagesWithPollingFragment$key } from '~/generated/useGetPreviewImagesWithPollingFragment.graphql';
```

Comment:
the other file with the important updates

this hook should almost always be used in conjunction with `NftFailureBoundary`